### DISCUSSION TITLE
using-mvp-to-develop-features/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#15] [Comments]
# using-mvp-to-develop-features/

In the last ~20 years, MVP releases have thankfully become the standard way of developing software. However, this mindset can be taken further, and used as a way to work on an individual ticket or feature, even as a solo dev. Why? Well, here’s a whole post full of motivations…

https://blog.jakelee.co.uk/using-mvp-to-develop-features/

**Comment by jackdann555** [comment]
It's a helpful explanation about the feature of <a href="https://10decoders.com/services/mvp-development/">MVP Development for Startups</a>. The way of you describing the process and the snapshot is useful to beginners like me.

Thank you for this blog Mr. Jake Lee
